pkgs/development/python-modules/gpt-2-simple/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ regex requests tqdm numpy toposort tensorflow ];
 
-  dontCheck = true; # no tests in upstream
+  doCheck = false; # no tests in upstream;
```

Comment:
A testing hook is automatically added when using the legacy `format = "setuptools"` way of packaging python packages. You can set `pyproject = true` instead of setting `format` and add `setuptools` to `nativeBuildInputs` (or `build-system`). This will use the `pypa` builder, which is the recommended way of doing this now.

If you do this, no disabling is needed, because we won't be including any test hooks, unlike when setting `format`

Reply:
IMO while you're at it, you should also do this for `kaitaistruct`, which also tries to run tests, but can't find any.

Reply:
Great feedback, @TomaSajt, I'll look into in the coming days. 👍

Reply:
@TomaSajt I've update this PR to also `doCheck = false;` for `kaitaistruct` as you recommended.